packages/paima-sdk/paima-utils/src/config/loading.ts

```diff
@@ -10,6 +10,8 @@ export enum ConfigNetworkType {
   EVM_OTHER = 'evm-other',
   CARDANO = 'cardano',
   MINA = 'mina',
+  AVAIL_MAIN = 'avail-main',
+  AVAIL_OTHER = 'avail-other',
```

Comment:
I think right now in the docs we don't talk about the avail-main option, right?

Reply:
talk in which way? it is documented as a possible value, and it's also documented that it's exclusive with `evm-main`, but maybe it's purpose it's not clear enough if you don't deduce what it does through the analogue with the `evm-main`.